### PR TITLE
Deadline and grace period updates

### LINKED ISSUE #264: Update for deadline & grace period interactions and deadline extension
see https://github.com/dobtco/screendoor-v2/pull/3180

### PR DESCRIPTION
**Pull request by kariannemah**
Closes #264.

### CODE REVIEW

File: contents/articles/screendoor/projects/deadline.md [hunk 1]
```diff
@@ -13,13 +13,13 @@ Under &ldquo;Deadline&rdquo; select a date and time to set a deadline for respon
 
 ![Setting a deadline.](../images/deadline_1.png)
 
-If you need to extend the due date later, return to this page and set a new date. To clear the due date, click &ldquo;Clear deadline&rdquo; below the date and time pickers.
+If you need to extend the due date later, return to this page and set a new date. Respondents who haven't yet submitted to your project will be notified of the updated due date. You can also [extend the deadline for a specific response](../responses/providing_support_to_respondents.html#extending-the-deadline-for-a-draft).
```
Comment (ajb) [inline contents/articles/screendoor/projects/deadline.md]: This isn't true. Respondents are only notified if _their_ deadline has been extended on the Unsubmitted Responses page.

File: contents/articles/screendoor/responses/providing_support_to_respondents.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ If you find that respondents are repeatedly getting stuck on a specific section
 
 #### Extending the deadline for a draft
 
-If someone can't complete your form by the [deadline](/articles/screendoor/projects/deadline.html) you have set and needs more time, you can extend the deadline for only that response. On the &ldquo;Unsubmitted responses&rdquo; page, press &ldquo;Extend&rdquo; next to their response in progress, and specify a new date and time.
+If someone can't complete your form by the [deadline](/articles/screendoor/projects/deadline.html) you have set and needs more time, you can extend the deadline for only that response. On the &ldquo;Unsubmitted responses&rdquo; page, press &ldquo;Extend&rdquo; next to their response in progress, and specify a new date and time. They will be notified via email of the new deadline.
```
Comment (ajb) [inline contents/articles/screendoor/responses/providing_support_to_respondents.md]: :+1:

Comment (kariannemah) [inline contents/articles/screendoor/responses/providing_support_to_respondents.md]: Got it. I was looking over the code and was _just_ about to ask you to clarify. I’ll update.


On February 9, 2016 at 11:26:22 AM, Adam Becker (notifications@github.com) wrote:

In contents/articles/screendoor/responses/providing_support_to_respondents.md:

**Comment by ajb** [comment]
Just realized this never got merged, it looks ready tho.

**Pull request merged**